src/arrow2/src/io/parquet/read/deserialize/binary/basic.rs

```diff
@@ -131,7 +131,7 @@ pub(super) type Dict = Vec<Vec<u8>>;
 
 #[derive(Debug)]
 pub(super) struct RequiredDictionary<'a> {
-    pub values: hybrid_rle::HybridRleDecoder<'a>,
+    pub values: hybrid_rle::HybridRleDecoder<'a, u32>,
```

Comment:
We can leave the Decoder hardcoded to u32 since I believe parquet will always give us that. No generics needed here

Reply:
Sounds good, undone!